FEATURES.md

```diff
@@ -2,14 +2,15 @@
 * Hide Messages: Ability Cooldown, Heal, AOTE, Implosion, Molten Wave, Teleport Pad Messages
 * Dungeon Minimap
 * Dungeon Puzzle Solver:
-    * Three Weirdos
-    * Blaze
-    * Croesus
-    * F7 Terminal: Order, Color, Name
+  * Three Weirdos
+  * Blaze
+  * Croesus
+  * F7 Terminal: Order, Color, Name
```

Comment:
Maybe drop the F7 since M7 also exists. I would also specify colour as "Coloured Items" because there's two terminals about coloured things and we only have a solver for one of them. Maybe also slightly expand on the name one too to be something like "Item Name"